src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_PrivacyScreen.tsx

```diff
@@ -34,39 +34,42 @@ export const Profile_PrivacyScreen = () => {
   const [isCleaningCollectedData, setIsCleaningCollectedData] =
     React.useState<boolean>(false);
   return (
-    <View style={style.container}>
-      <FullScreenHeader
-        title={t(ProfileTexts.sections.privacy.heading)}
-        leftButton={{type: 'back'}}
-      />
-      <ScrollView>
+    <FullScreenView
+      headerProps={{
+        title: t(ProfileTexts.sections.privacy.heading),
+        leftButton: {type: 'back'},
+      }}
+    >
+      <ScrollView contentContainerStyle={style.content}>
```

Comment:
There is already a `ScrollView` inside `FullScreenView`, maybe try making this just a `View`?

Reply:
Thanks! Didn't think of that.